api/src/tauri.ts

```diff
@@ -67,6 +70,17 @@ async function invoke<T>(args: any): Promise<T> {
       Reflect.deleteProperty(window, callback)
     }, true)
 
+    if (typeof cmd === 'string') {
+      args.cmd = cmd
+    } else if (typeof cmd === 'object') {
+      if (args !== {}) {
+        console.log(cmd, args)
```

Comment:
console.log('oops')

Reply:
Lol good catch, can't believe I missed that :facepalm:.